spec/system/admin/subscriptions/helper.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'system_helper'
+
+def fill_in_subscription_basic_details
+  select2_select customer.email, from: "customer_id"
+  select2_select schedule.name, from: "schedule_id"
+  select2_select payment_method.name, from: "payment_method_id"
+  select2_select shipping_method.name, from: "shipping_method_id"
+
+  find_field("begins_at").click
+  choose_today_from_datepicker
+end
```

Comment:
These methods are now defined at a global level. It's better to wrap them in a module and include the module where you need it. That's how the other helper files work.

Reply:
Thanks, this looks like a minor comment, but really helped me understand how this is optimally set :pray: